### PR TITLE
[libbeat]add EC2 fields to yaml for add_cloud_metadata

### PR DESCRIPTION
**Pull request by odacremolbap**
Related to https://github.com/elastic/beats/pull/12307
As mentioned here https://github.com/elastic/beats/pull/12307#issuecomment-496820481

Adding fields to yaml file for EC2 cloud metadata

@exekias

### CODE REVIEW

File: libbeat/processors/add_cloud_metadata/_meta/fields.yml [hunk 1]
```diff
@@ -44,3 +49,14 @@
       type: alias
       path: cloud.region
       migration: true
+
+    - name: meta.cloud.account.id
```
Comment (ruflin) [inline libbeat/processors/add_cloud_metadata/_meta/fields.yml]: Do not have the full context on this one but I wonder why `migration: true` is missing?

Comment (exekias) [inline libbeat/processors/add_cloud_metadata/_meta/fields.yml]: These new alias are not needed. They were initially created for backwards compatibility with previous names. As these fields are new, we don't need to have them in the old layout

Comment (exekias) [reply libbeat/processors/add_cloud_metadata/_meta/fields.yml]: I submitted another comment with the review :innocent:, I guess we don't need these alias cause they are primary minded for backwards compatibility, but these are new fields

Comment (odacremolbap) [reply libbeat/processors/add_cloud_metadata/_meta/fields.yml]: I copied it from existing fields.yml

I though that those fields needed to be defined at the event, and then aliased to ECS.
are there some docs on how to use this fields.yml at processors and how it fits with ECS?

Comment (odacremolbap) [reply libbeat/processors/add_cloud_metadata/_meta/fields.yml]: I thought data from the processor was created under `meta.cloud ...` and then had to be aliased to ECS `cloud. ...`

idk how this works, but sounds like I just need to remove this block from fields.yml, and that _somehow_ the field will be mapped to ECS

Comment (exekias) [reply libbeat/processors/add_cloud_metadata/_meta/fields.yml]: Sorry I don't think there are docs about this. These aliases were created in the context of ECS transition (see https://github.com/elastic/beats/pull/9265 for more details). In a nutshell, old fields were not ECS compliant, so everything was moved to ECS fields, then old fields became aliases pointing to the new ones, as a way to allow smoother upgrades for users.

As you are adding new fields, there is no need to create alias from old names, as nobody is relying on these names for their visualizations

**Comment by odacremolbap** [comment]
not sure if something is missing:
- added `cloud.image.id` since it is not part of ECS
- removed previously added at this PR `cloud.account.id`, being part of ECS

**Pull request merged**